packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "flowise-components",
-    "version": "2.2.1",
+    "version": "2.2.2",
```

Comment:
We don't have to upgrade the dependency in this PR, will do it on separate one

Reply:
Thank you for your feedback. I restored the version to 2.2.1 then.